packages/components/src/utils/number.util.ts

```diff
@@ -1,20 +1,23 @@
-import { localstring } from './local.util'
+import { useBalConfig } from '../config'
 
 export const getDecimalSeperator = (): string => {
-  return Intl.NumberFormat(localstring())
+  const config = useBalConfig()
+  return Intl.NumberFormat((config && config.locale) || 'de-CH')
```

Comment:
'de-CH' -> maybe we can switch this to some constant so we do not have magic one or if we have this config locals somewhere ?

Reply:
thanks adjust that with the default values